shared/chat/conversation/messages/wrapper/exploding-meta.js

```diff
@@ -207,11 +209,44 @@ const styles = styleSheetCreate({
       width: 55,
     },
   }),
-  countdownContainer: {
-    borderRadius: 2,
-    paddingLeft: 4,
-    paddingRight: 4,
-  },
+  countdown: platformStyles({
+    common: {color: globalColors.white, fontSize: 10, fontWeight: 'bold'},
+    isElectron: {lineHeight: '1.4em'},
```

Comment:
youcan just put lineHeight in common and it'll do teh right thign in platform styles